### ISSUE TITLE
Does your code pass the "abducted by aliens" test? 

### ISSUE THREAD

**Issue by yvonne-liu** [FAC10/week3-PAMF#80]
We're struggling to follow along with your code, so maybe a good idea would be to add more comments and/or documentation please! ✨ ✨ ✨

**Comment by oliverjam** [comment]
Yeah I am struggling a bit reviewing this because I can't really follow exactly where stuff is happening as the code jumps back and forth from each file. This may just be because I don't understand Waterfall properly though!
